Vagrantfile

```diff
@@ -0,0 +1,76 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+VIRTUAL_MACHINES = {
+  :trusty => {
+    :hostname => 'st2-packages-trusty',
+    :box => 'ubuntu/trusty64',
+    :ip => '192.168.16.20',
+  },
+  :xenial => {
+    :hostname => 'st2-packages-xenial',
+    :box => 'ubuntu/xenial64',
+    :ip => '192.168.16.21',
+  },
+  :el7 => {
+    :hostname => 'st2-packages-el7',
+    :box => 'centos/7',
+    :ip => '192.168.16.22',
+  },
+}
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  VIRTUAL_MACHINES.each do |name, cfg|
+    config.vm.define name do |vm_config|
+      vm_config.vm.hostname = cfg[:hostname]
+      vm_config.vm.box = cfg[:box]
+
+      # Give VM access to all CPU cores on the host
+      # docker-compose & rake build can benefit from more CPUs
```

Comment:
I added CPUs autodetection to use max on guest VM.
That's pretty important, since that ruby Rake thing in docker-compose build can benefit from it to speed up the build.

See: 
![screenshot from 2017-01-25 19-10-24](https://cloud.githubusercontent.com/assets/1533818/22301194/35cad708-e33b-11e6-8f16-a246c628c5f8.png)